code/modules/modular_computers/file_system/programs/ntmessenger.dm

```diff
@@ -146,44 +187,43 @@
 				to_chat(usr, span_notice("ERROR: Device does not have mass-messaging perms."))
 				return
 
-			var/list/targets = list()
-
-			for(var/obj/item/modular_computer/mc in GetViewableDevices())
-				targets += mc
+			if(can_send_everyone_message())
+				to_chat(usr, span_warning("The subspace transmitter of your tablet is still cooling down!"))
```

Comment:
Do you think it would be better if this was a TGUI thing? Like, having the 'send to all' button greyed out with a tooltip saying this instead.
Up to you though since this is not part of your PR technically.

Reply:
I was gonna add that, the PR is not feature complete yet